src/platforms/javascript/guides/nextjs/manual-setup.mdx

```diff
@@ -194,6 +194,8 @@ By default, `withSentryConfig` will add an instance of `SentryWebpackPlugin` to
 
 To configure the plugin, pass a `SentryWebpackPluginOptions` argument to `withSentryConfig`, as seen in the example above. All available options are documented [here](https://github.com/getsentry/sentry-webpack-plugin#options).
 
+### Disable `SentryWebpackPlugin`
```

Comment:
Not sure if the same capitalization rules apply here as with your other PR, but wondering if SentryWebpackPlugin should be in camel case here as well?

Reply:
Not in this case, `PascalCase` is correct as this is how user-facing object is exported :)

Reply:
Yeah, here we're talking about the actual plugin (which is a class, hence written in upper camel case, a.k.a. Pascal case), whereas the places that I fixed were a variable pointing to an options object for that plugin.